### PR TITLE
add save selected samples

### PR DESCRIPTION
**Pull request by Jade-Codes**
### What does this PR do?
Add's saved selected samples

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### CODE REVIEW

File: client/src/pages/recording-view/components/settings-pane.tsx [hunk 1]
```diff
@@ -57,19 +57,19 @@ const SettingsPane = ({ currentFFT }) => {
...
-    // TODO: Return with the download of the blob
-    // var blobUrl = window.URL.createObjectURL(blob);
-    // var blob = new Blob([trimmedSamples], { type: 'octet/stream' });
-    // a.href = blobUrl;
-    // a.download = 'trimmedSamples.sigmf-data';
-    // a.click();
-    // window.URL.revokeObjectURL(blobUrl);
+    // Return with the download of the blob
+    const blobUrl = window.URL.createObjectURL(
+      new Blob([cursorContext.cursorData], { type: 'application/octet-stream' })
```
Comment (Nepomuceno) [inline client/src/pages/recording-view/components/settings-pane.tsx:67]: Just because I was thinking on removing cursor data LOL

**Review by Nepomuceno — COMMENTED** [review]
image

**Review by Nepomuceno — APPROVED** [review]

File: client/src/pages/recording-view/components/settings-pane.tsx (hunk 1 above)
Comment (Jade-Codes) [reply client/src/pages/recording-view/components/settings-pane.tsx:67]: image

**Review by hannahlspencer — APPROVED** [review]
lgtm!

**Pull request merged**